package.json

```diff
@@ -8,11 +8,11 @@
     "build": "rimraf lib && babel src -d lib",
     "lint": "eslint src test",
     "pretest": "npm run --silent lint",
-    "test": "npm run build && npm run test:coverage && npm run --silent cover:check",
-    "tests-only": "npm run build && npm run test:quick",
-    "test:coverage": "babel-node node_modules/.bin/istanbul cover --report html node_modules/.bin/_mocha -- -R tap test/init.js test/*-test.js",
-    "cover:check": "istanbul check-coverage && echo code coverage thresholds met, achievement unlocked!",
-    "test:quick": "babel-node node_modules/.bin/_mocha -R tap test/init.js test/*-test.js"
+    "test": "npm run test:coverage && npm run --silent cover:check",
+    "tests-only": "npm run test:quick",
+    "test:coverage": "nyc --silent node_modules/.bin/_mocha -R tap test/init.js test/*-test.js",
```

Comment:
while we're at it, can this just use `mocha` directly, instead of needing the `_mocha` istanbul hack?